src/main/java/sparksoniq/io/shell/JiqsJLineShell.java

```diff
@@ -110,8 +111,10 @@ private void initialize() throws IOException {
         Terminal terminal = TerminalBuilder.builder()
                 .system(true)
                 .build();
+        DefaultParser parser = new DefaultParser();
+        parser.setEscapeChars(null);
```

Comment:
Deactivate \ escaping on JLine so Apache Commons can do so instead.
Version needed to be updated to 3.11.0 because of a known bug.